web/messages/ar.json

```
@@ -1008,6 +1017,38 @@
           "globalFontMagnification": "يضبط بهذا المقدار النسبي أحجام الخطوط في الطرفيات وعناوين التبويبات والأشرطة الجانبية والإعدادات والتراكبات وواجهة التطبيق التي يملكها cmux. لا يشمل محتوى صفحات المتصفح المعروض."
         },
         "terminal": {
+          "resumeCommands": {
```

Biome (2.5.6)

[error] The key resumeCommands was already declared.

(lint/suspicious/noDuplicateObjectKeys)